02_tools-and-packages/05_sklearn.ipynb

```diff
@@ -179,7 +179,7 @@
     }
    },
    "source": [
-    "Always **reserve some data for testing** your algorithm. There are many algorithms which suffer from overfitting. Hence, test the trained algorithm on some unseen data. In practice you can use cross validation to determine the performance of your algorithm."
+    "Always [reserve some data for testing](https://scikit-learn.org/stable/modules/cross_validation.html) your algorithm. There are many algorithms which suffer from overfitting. Hence, test the trained algorithm on some unseen data. In practice you can use cross validation to determine the performance of your algorithm."
```

Comment:
Could we use a bold font here?

```suggestion
    "Always [**reserve some data for testing**](https://scikit-learn.org/stable/modules/cross_validation.html) your algorithm. There are many algorithms which suffer from overfitting. Hence, test the trained algorithm on some unseen data. In practice you can use cross validation to determine the performance of your algorithm."
```

Reply:
Of course. I tried the `**` around the brackets `[ ]` and it didn't work. :-)